### ISSUE TITLE
iOS 13 scrolling on panmove

### ISSUE THREAD

**Issue by Jenq** [hammerjs/hammer.js#1240]
Good day to all!
Faced a problem that the scroll works on panmove event in iOS 13, but on lower versions all works as expected (scroll is disabled wile pan is running). Nevertheless, if you allow the scroll to work together with the panmove, the panend will not triggered.
Any ideas what I'm doing wrong or this is really bug?
[Stackblitz example](https://stackblitz.com/edit/angular-material-fb9lgy)

**Comment by erdeszt** [comment]
Have you checked if this fix works in your case: https://github.com/hammerjs/hammer.js/issues/1084#issuecomment-544220158 ?

**Comment by Jenq** [comment]
> Have you checked if this fix works in your case: [#1084 (comment)](https://github.com/hammerjs/hammer.js/issues/1084#issuecomment-544220158) ?

Yes, it's works, but only for the second case. Thanks for this, I don't know how I missed this comment.
